src/scripts/questionnaire.js

```diff
@@ -384,6 +432,14 @@ export default class Questionnaire extends H5P.EventDispatcher {
     this.attach = function ($wrapper) {
       $wrapper.get(0).classList.add('h5p-questionnaire-wrapper');
       $wrapper.get(0).appendChild(questionnaireWrapper);
+
+      if(this.state.finished && !behaviouralSettings.enableEditRights) {
```

Comment:
Nit: inconsistent formatting

Can we share the finished if statement with the one below?